adafruit_rgb_display/st7735.py

```diff
@@ -97,12 +100,12 @@ class ST7735(DisplaySPI):
     )
     _ENCODE_PIXEL = ">H"
     _ENCODE_POS = ">HH"
-
-    def __init__(self, spi, dc, cs, rst=None, width=128, height=128):
-        super().__init__(spi, dc, cs, rst, width, height)
+    #    def __init__(self, spi, dc, cs, rst=None, width=1, height=1, baudrate=1000000,
+    #                 polarity=0, phase=0)
 
 
 class ST7735R(ST7735):
```

Comment:
Tested with ST7735R display, shows bars and multicolored pixellation across the display. Tried with example code included at beginning of file, as well as other code. Both of my examples run with the current version (if I change the last line to `self.write` in the current version as well).